### DISCUSSION TITLE
[bus error] device status: eBUS comm error: overrun

### DISCUSSION THREAD

**Discussion by BlackOrca** [john30/ebusd#1015] [Q&A]
What dos the log entry mean?
What options can i adjust to get away this message. I think that is not a good entry!

Br,
Dennis

**Comment by olli-r** [comment]
I have the same message:

`2023-10-23 09:18:56.110 [bus error] device status: eBUS comm error: overrun`

As well as lots of "ERR: wrong symbol recieved".

How can I debug this further?

- ebus Adapter 5
- ebusd running in docker container, connecting via network in enhanced mode

Thanks,

Olli

**Comment by ChrisMuc** [comment]
same log message here; and only some basic values are published to MQTT broker.
- ebus Adapter 5, Build: [20231015](https://adapter.ebusd.eu/v5/ChangeLog#20231015)
- ebusd running in docker container, connecting via network (`ens:IP:9999`)
- `[bus notice] scan 08: ;Vaillant;BAI00;0518;7401` and `[bus notice] scan 15: ;Vaillant;43000;0215;2002`

**Comment by GitSoDE** [comment]
Did you find a solution ?

**Reply by ChrisMuc** [reply]
I did not investigate it further so far, as ebusd is running and communication is working anyways. I can query all values and get them returned. My guess: the error is just related to arbitration during initialization (and maybe later when polling values).

By default, i.e. when using unmodified configuration files (e.g. automatically) downloaded from github, only basic values are published to MQTT. If you want to receive more values, you need to query/poll them.

According to some comments I read, there is an option to download and adjust the configuration files for your heater (or whatever your ebus is connected to). As far as I remember, you can change the value in the first column, so that the selected values are polled and published automatically by ebusd. But I did not try this. I'm using Node-Red to periodically poll the values I'm interested in. That has the advantage that I can change the frequency in Node-Red without restarting (or reloading) ebusd.
